airbyte-ci/connectors/connector_ops/connector_ops/acceptance_test_config_checks.py

```diff
@@ -58,21 +59,27 @@ def get_bypass_reason_changes() -> Set[utils.Connector]:
     return bypass_reason_changes.intersection(find_changed_important_connectors())
 
 
-def find_mandatory_reviewers() -> List[Union[str, Dict[str, List]]]:
+def find_mandatory_reviewers() -> List[Dict[str, Union[str, Dict[str, List]]]]:
     important_connector_changes = find_changed_important_connectors()
     backward_compatibility_changes = utils.get_changed_acceptance_test_config(diff_regex="disable_for_version")
     test_strictness_level_changes = utils.get_changed_acceptance_test_config(diff_regex="test_strictness_level")
     ga_bypass_reason_changes = get_bypass_reason_changes()
+    breaking_change_changes = utils.get_changed_metadata(diff_regex="upgradeDeadline")
+
+    required_reviewers = []
 
     if backward_compatibility_changes:
-        return [{"any-of": list(BACKWARD_COMPATIBILITY_REVIEWERS)}]
+        required_reviewers.append({"name": "Backwards Compatibility Test Skip", "teams": list(BACKWARD_COMPATIBILITY_REVIEWERS)})
     if test_strictness_level_changes:
-        return [{"any-of": list(TEST_STRICTNESS_LEVEL_REVIEWERS)}]
+        required_reviewers.append({"name": "Acceptance Test Strictness Level", "teams": list(TEST_STRICTNESS_LEVEL_REVIEWERS)})
     if ga_bypass_reason_changes:
-        return [{"any-of": list(GA_BYPASS_REASON_REVIEWERS)}]
+        required_reviewers.append({"name": "GA Acceptance Test Bypass", "teams": list(GA_BYPASS_REASON_REVIEWERS)})
     if important_connector_changes:
-        return list(GA_CONNECTOR_REVIEWERS)
-    return []
+        required_reviewers.append({"name": "GA Connectors", "teams": list(GA_CONNECTOR_REVIEWERS)})
+    if breaking_change_changes:
+        required_reviewers.append({"name": "Breaking Changes", "teams": list(BREAKING_CHANGE_REVIEWERS)})
+
+    return required_reviewers
```

Comment:
This used to have early returns, which meant cases where multiple requirements are involved didn't work. Instead, we're adding multiple requirement groups that all have to pass. The contents of this dict is based on the format laid out in https://github.com/Automattic/action-required-review#requirements-format

As seen in the [example](https://github.com/Automattic/action-required-review#example-1), they list out multiple review groups that each have a name and team list.